src/commands/create.js

```diff
@@ -150,5 +149,27 @@ export default asyncCommand({
 	}
 })
 
+const install = async (cwd, packages, env) => {
+	const isDev = env === 'dev' ? true : false
+	const isYarnAvailable = await isCommandAvailable('yarn')
 
-const npm = (cwd, args) => spawn('npm', args, { cwd, stdio: 'ignore' });
+	if(isYarnAvailable) {
+		const args = ['add']
+		if(isDev) {
+			args.push('-D')
+		}
+
```

Comment:
We can slim this down by moving the `.filter(Boolean)` into install, but I'll just slip that in post-merge :)